### PR TITLE
AUT-131_part1

### PR DESCRIPTION
**Pull request by azayatsfv**
GET add_company
GET get_companies
GET get_companies_by_ids

GET get_company_ids
GET get_company_license_types
GET get_company_logo

### CODE REVIEW

File: APIs/companies/requests/get_companies_by_ids.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from dataclasses import dataclass
+from base.common.models.request import BaseRequestModelKeys, SimpleRequestModel
+
+
+@dataclass
+class GetCompaniesByIDsParams(BaseRequestModelKeys):
+    COMPANY_IDS	: str = "CompanyIds"
```
Comment (darkhipenko) [inline APIs/companies/requests/get_companies_by_ids.py:7]: extra space

**Review by darkhipenko — CHANGES_REQUESTED** [review]
+ responses/get_company_logo.py (empty file)

File: APIs/companies/requests/get_companies_by_ids.py (hunk 1 above)
Comment (azayatsfv) [reply APIs/companies/requests/get_companies_by_ids.py]: fixed

**Review by darkhipenko — APPROVED** [review]
Approved

**Pull request merged**